e2eAIOK/DeNas/search.py

```diff
@@ -59,6 +62,15 @@ def main(params):
         super_net = TransformerASRSuper
         search_space = {'num_heads': cfg.SEARCH_SPACE.NUM_HEADS, 'mlp_ratio': cfg.SEARCH_SPACE.MLP_RATIO,
                         'embed_dim': cfg.SEARCH_SPACE.EMBED_DIM , 'depth': cfg.SEARCH_SPACE.DEPTH}
+    elif params.domain == 'thirdparty':
```

Comment:
replace 'thirdparty' with 'huggingface' since we not actually support "thirdparty"